cumulusci/tasks/util.py

```diff
@@ -70,36 +80,47 @@ def _run_task(self):
 
 class Sleep(BaseTask):
     name = "Sleep"
-    task_options = {
-        "seconds": {"description": "The number of seconds to sleep", "required": True}
-    }
+
+    class Options(CCIOptions):
+        seconds: int = Field(..., description="The number of seconds to sleep")
+
+    parsed_options: Options
```

Comment:
Is there a reason we see this line on some of the refactored tasks and not others?

Reply:
I don't think it is needed. Probably just for type checking if we ever enable mypy.